pyproject.toml

```diff
@@ -68,3 +68,8 @@ strict_equality = true
 [[tool.mypy.overrides]]
 module = ["argcomplete"]
 ignore_missing_imports = true
+
+[tool.ruff]
+target-version = "py39"
+line-length = 119
```

Comment:
I think we want 109 which is what systemd uses as well

Reply:
I'm open to that. The first iteration of this actually had 109, but I changed it to 119, since that was the previous max for black and flake8 before that. It's just many more instances and some are a bit awkward.

Reply:
Right let's keep this for now then, we can always lower it later

Reply:
Dropped the commit again.